graph/package.json

```diff
@@ -20,6 +20,8 @@
     "create-local": "graph create --node http://localhost:8020/ hypercerts-admin/hypercerts-testnet",
     "remove-local": "graph remove --node http://localhost:8020/ hypercerts-admin/hypercerts-testnet",
     "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 hypercerts-admin/hypercerts-testnet",
+    "create_with_env": "graph create --node $GRAPH_URL $SUBGRAPH_NAME",
```

Comment:
Ah ya this would be generally part of the docker compose process. I wouldn't expect it to be something you could interact with. I wish I could have an inline comment, but alas, JSON.

Reply:
The other thing I _could_ do is kinda like the `hardhat` solution [above](https://github.com/hypercerts-org/hypercerts/pull/855#discussion_r1234870732). This is the same problem that it solves actually.

Reply:
Actually ya, I got rid of this in favor of using something similar to the hardhat solution.